statsmodels/discrete/discrete_model.py

```diff
@@ -1308,7 +1334,7 @@ def hessian_factor(self, params):
         exposure = getattr(self, "exposure", 0)
         X = self.exog
         L = np.exp(np.dot(X,params) + exposure + offset)
-        return L
+        return -L
```

Comment:
why does this change the sign?
(it's in one of the merge conflicts, and I'm not sure what's correct)